server/ec2spotmanager/tasks.py

```diff
@@ -25,7 +26,7 @@
 
 @app.task
 def check_instance_pool(pool_id):
-    from .models import Instance, InstancePool, INSTANCE_STATE, PoolStatusEntry, POOL_STATUS_ENTRY_TYPE
+    from .models import Instance, InstancePool, EC2_INSTANCE_STATE, PoolStatusEntry, EC2_POOL_STATUS_ENTRY_TYPE
```

Comment:
These should be abstracted in the provider interface.

Reply:
This was moved to the provider interface.